docs/pharmaceutical-treatment.rst

```diff
@@ -0,0 +1,82 @@
+.. _rstpharmaceuticaltreatment:
```

Comment:
I thought this was just "Treatment"? Where does the "Pharmaceutical" come from?

Reply:
The v2 branch is 57 commits behind master - v2 needs deleting!  I've opened a new PR with the pedigree changes, including updates from your suggestions here - https://github.com/phenopackets/phenopacket-schema/pull/364